static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -241,186 +231,48 @@ class InviteMembersModal extends DeprecatedAsyncComponent<
   }
 
   render() {
-    const {Footer, closeModal, organization} = this.props;
-    const {pendingInvites, sendingInvites, complete, inviteStatus, member} = this.state;
-
-    const disableInputs = sendingInvites || complete;
```

Comment:
This one variable is moved into the new file. It's the only one not passed directly through.